### PR TITLE
server: Implement issuance function

### LINKED ISSUE #50: Implement the function to issue properties from the server
## Feature Request

**Is your feature request related to a problem? Please describe.**  

Currently, only the function of signing the value sent by the user is supported.

**Describe the solution you'd like**  

A function to issue the authenticated value in the server must be implemented.

**Describe alternatives you've considered**  

**Additional context**

### PR DESCRIPTION
**Pull request by HyeockJinKim**
# Summary 

Implement function for use when server issues properties.

- It closes #50

## How to Review? 

- Please refer to my self-review submitted as the review.

### CODE REVIEW

File: packages/server/src/index.ts [hunk 1]
```diff
@@ -89,6 +96,28 @@ export function verifyPackage(
...
+  switch (type) {
+    case AuthType.Privacy:
+      return verifyPrivacy(properties, sign, publicKey);
+    case AuthType.Package:
+      return verifyPackage(properties, sign, publicKey);
+    default:
+      throw new Error('Unreachable Code');
+  }
+}
+
```
Comment (HyeockJinKim) [inline packages/server/src/index.ts:120]: refactoring.

File: packages/server/src/index.ts [hunk 2]
```diff
@@ -147,3 +160,12 @@ export function verifyProperty<T>(
...
   };
 }
+
+export function issueProperty<T>(
+  object: { [key: string]: string },
+  secret: SecretKey,
+  authType: AuthType,
+): Signature {
+  const properties = objectToProperty(object);
+  return authByAuthType(properties, secret, authType);
```
Comment (HyeockJinKim) [inline packages/server/src/index.ts:170]: Receives an object as an input and creates a property array and then authenticates this properties.
A function to issure properties from server.

File: packages/server/src/utils.ts [hunk 3]
```diff
@@ -92,3 +92,11 @@ export function propertyObject(properties: Property[]): { [key: string]: string
...
   );
 }
+
+export function objectToProperty(object: { [key: string]: string }): Property[] {
+  return Object.keys(object).map(key => ({
+    type: PropertyType.Raw,
+    key,
+    value: object[key]
+  }))
+}
```
Comment (HyeockJinKim) [inline packages/server/src/utils.ts:102]: simply make properties using object.

**Review by HyeockJinKim — COMMENTED** [review]
self review

**Review by anthonyminyungi — APPROVED** [review]
We've discussed it in person.
Looks good.

**Pull request merged**